uwsgi.ini

```diff
@@ -25,10 +25,24 @@ vacuum = True       # Try to remove all of the generated files/sockets (UNIX soc
 module =kolibri.deployment.default.wsgi:application
 buffer-size = 8192  # needed to support long requests blocks (in lessons, for example)
 wsgi-disable-file-wrapper = true  # needed to fix uwsgi bug https://github.com/unbit/uwsgi/issues/1126
+single-interpreter = true
+die-on-term = true                   # Shutdown when receiving SIGTERM (default is respawn)
+need-app = true
+
+disable-logging = true               # Disable built-in logging
+log-4xx = true                       # but log 4xx's anyway
+log-5xx = true                       # and 5xx's
```

Comment:
This is because we don't have log rotation?

Reply:
No, it's to avoid duplication. The same info is available in the nginx logs, excepting the python errors.